opentech/apply/determinations/templates/determinations/determination_detail.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base-apply.html" %}
+{% load bleach_tags %}
+
+{% block content %}
+<div class="wrapper wrapper--breakout wrapper--admin">
+    <div class="wrapper wrapper--large">
+        <h2 class="heading heading--no-margin">Determination</h2>
+        <h5>For <a href="{% url "funds:submissions:detail" determination.submission.id %}">{{ determination.submission.title }}</a></h5>
+    </div>
+</div>
+
+<div class="grid">
+    <div>
+        <h5>Determination</h5>
+        <p>{{ determination.get_determination_display }}</p>
+        {{ determination.determination_message|bleach }}
+    </div>
+</div>
+
+    {% if can_view_extended_data %}
```

Comment:
If we are using the same pattern as earlier we should probably be providing two views. One for admin and one for applicant and non privileged staff.